src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -561,6 +561,43 @@ public void TestDecryptSimple_Pkcs7Signed_GeneratedOnLinux()
             VerifySimpleDecrypt(encodedMessage, certLoader, expectedContentInfo);
         }
 
+        [Fact]
+        [OuterLoop(/* Leaks key on disk if interrupted */)]
+        public void TestDecryptSimpleAes256_RsaOaepSha256()
+        {
+            byte[] encodedMessage = Convert.FromBase64String(
```

Comment:
The openssl command used to create a CMS with RSAES-OAEP-SHA256 is something like this:

```
openssl cms -encrypt -in input.txt -out out.msg -recip cert.pem -keyopt rsa_padding_mode:oaep -keyopt rsa_mgf1_md:sha256 --aes256 -keyopt rsa_oaep_md:sha256
```

Reply:
Might be worth putting that as a comment.